sbpy/activity/gas/core.py

```diff
@@ -635,3 +638,477 @@ def total_number(self, aper, eph=None):
+                    then jumps up to 5e29 at 15 days ago until the current (observation) time.
+                        ---
+                        inputParameters['ProductionRates'] = [1.e28, 5.e29]
+                        inputParameters['TimeAtProductions'] = [30, 15] * u.day
+                        ---
+            --- Parent ---
+            inputParameters['Parent']['TotalLifetime']: Total lifetime of parent molecule, with units
+            inputParameters['Parent']['DissociativeLifetime']: Dissociative lifetime of parent molecule, with units
+            inputParameters['Parent']['Velocity']: Outflow velocity of parent molecules, with units
+            (Optional) inputParameters['Parent']['CrossSection']: Cross section of parent molecule
+            (Optional) inputParameters['Parent']['DestructionLevel']: Fraction of destroyed parent molecules at which we
+                stop tracking them
+
+            --- Fragment ---
+            inputParameters['Fragment']['TotalLifetime']: Total lifetime of fragment molecule, with units
+            inputParameters['Fragment']['Velocity']: Velocity of fragment after dissociative kick, with units
+            (Optional) inputParameters['Fragment']['DestructionLevel']: Similar to above, but for the fragments
+
+            --- (Optional) Grid Specifications ---
+            inputParameters['Grid']['NumRadialGridpoints']: Number of radial points to calculate volume/column density
+            inputParameters['Grid']['NumAngularGridpoints']: Number of angular slices for volume density grid
+            inputParameters['Grid']['NumSubgridRadialSteps']: How finely to integrate radially for a given gridpoint
+            inputParameters['Grid']['NumSubgridAngularSteps']: How finely to integrate over ejection angle
+
+            --- (Optional) Printing ---
+            inputParameters['PrintDensityProgress']: Print progress while calculating volume density, if you're using a
+                large grid
+
+            References:
+                 The density distribution of neutral compounds in cometary atmospheres. I - Models and equations,
+                 Festou, M. C. 1981, Astronomy and Astrophysics, vol. 95, no. 1, Feb. 1981, p. 69-79.
+    """
+
+    """ NOTE: We allow Q to vary with time, so we pass it along to the base class but it is ultimately ignored in favor
+        of the production information coming in through inputParameters.  Likewise, v is also ignored because the parent
+        outflow velocity is included in inputParameters.
+        Calculations are done internally in meters and seconds to match the base GasComa class.
+    """
+    # TODO: We would have to rewrite the time-varying production stuff to accept Q here and then let the param structure
+    # define some variation of this initial production rate
+    @bib.cite({'model': '1981A&A....95...69F'})
+    @u.quantity_input(Q=(u.s**-1, u.mol / u.s), v=u.m / u.s)
+    def __init__(self, Q, v, inputParameters):
```

Comment:
There are a lot of options, but I think we can unpack them from the dictionary and leave them as individual keyword arguments.  Let me propose the following:

```
def __init__(self, Q, dt, parent, fragment, radial_points=X, radial_substeps=Y, angular_points=U, angular_substeps=V):
```
Then `parent` and `fragment` can use our ``Phys`` data class:
```python
parent = Phys.from_dict({
    tau_T=100 * u.day,
    tau_d=4e5 * u.s,
    v=1 * u.km / u.s,
    sigma=1e-10 * u.cm**2
})
```
Destruction level sounds like a model parameter, rather than a physical property, so perhaps it should also be a keyword argument to `__init__()`.